docs/iris/src/whatsnew/contributions_3.0.0/bugfix_2019-Nov-21_cell_measure_attributes.txt

```diff
@@ -0,0 +1,2 @@
+* Fixed a bug where the attributes of cell methods in netcdf-CF files were discarded on
+  loading.  They now appear on the CellMethod in the loaded cube.
```

Comment:
I believe this should refer to cell measures/CellMeasure rather than cell methods/CellMethod.

Reply:
Whoops! 
I can't believe I'm _still_ getting these names mixed up.
Will fix...